dev/acf-export.json

```diff
@@ -738,5 +727,47 @@
         "hide_on_screen": "",
         "active": true,
         "description": ""
+    },
+    {
+        "key": "group_5e8777494f9e9",
+        "title": "Post Fields",
+        "fields": [
+            {
+                "key": "field_5e87774dce0c5",
+                "label": "Deck",
+                "name": "post_deck",
+                "type": "wysiwyg",
```

Comment:
We may want to consider simplifying the WYSIWYG editing tools for this field--we probably don't want images and such to be added here.  I believe Today is using a custom WYSIWYG toolbar set for their "deck" field: https://github.com/UCF/Today-Child-Theme/blob/master/includes/config.php#L233-L249

Reply:
![Screen Shot 2020-04-06 at 10 58 20 AM](https://user-images.githubusercontent.com/1830748/78572586-ab214a80-77f5-11ea-947f-bf23d119b4c4.jpg)

These are the controls currently available. Do we need to reduce it down further? Images currently can't be uploaded.